### ISSUE TITLE
Console UI jumps to top of page when you click on a build

### ISSUE THREAD

**Issue by uglycoyote** [buildbot/buildbot#3657]
If the console view is long enough that you have to scroll down, when I click on a bubble (to view a build summary) it automatically scrolls back to the top of the console table.  This is annoying if you want to view a bunch of summaries that are all side by side but somewhere way down the page.

**Comment by uglycoyote** [comment]
This issue goes away if you remove the css rule that sets `height: 100%` on `body`.

I got this hint from (this thread)[https://github.com/angular-ui/bootstrap/issues/1016]

It is not clear to me what the purpose of setting `html` and `body` to have a height of 100%, shouldn't they just be as tall as the content that they contain?    Perhaps this is some semi-standard css trick copied from bootstrap somewhere?  I gathered that it may have something to do with making "sticky footers" work, but as far as I know those are not used anywhere in buildbot's UI.  I didn't see any problems so far running without the `height: 100%`.  Will propose a pull request with this change.
